src/opnsense/service/templates/OPNsense/Proxy/squid.user.local_auth.conf

```diff
@@ -1,5 +1,20 @@
+{% set voucher_method = [] %}
```

Comment:
it's probably better to move this a new file using  https://github.com/opnsense/core/blob/stable/17.1/src/opnsense/service/templates/OPNsense/Proxy/squid.conf#L278 , something like  /usr/local/etc/squid/post-auth/local_voucher.conf

To make it more explicit where the feature is for, if I'm seeing things correctly it only tries to set "auth_param basic casesensitive on" when a voucher provider is used.